packages/react-native-views/src/components/CameraView/hooks/useFakeActivity.js

```diff
@@ -0,0 +1,23 @@
+import { useCallback, useState } from 'react';
+import noop from 'lodash.noop';
+
+function useFakeActivity() {
+  const [fakeActivity, setFakeActivity] = useState(null);
+
+  const handleFakeActivity = useCallback((onEnd = noop) => {
+    const fakeActivityId = setTimeout(() => {
+      setFakeActivity(null);
+      onEnd();
+    }, 500);
```

Comment:
Why 0.5 seconds ? This should probably have a variable name and be just after the imports.

Reply:
This is a correct delay to see something is working, but if it's working too fast, the user will not see a flashing indicator.
Useful when we need to async check device features.

> 1 second: Limit for users feeling that they are freely navigating the command space without having to unduly wait for the computer. A delay of 0.2–1.0 seconds does mean that users notice the delay and thus feel the computer is "working" on the command, as opposed to having the command be a direct effect of the users' actions. Example: If sorting a table according to the selected column can't be done in 0.1 seconds, it certainly has to be done in 1 second, or users will feel that the UI is sluggish and will lose the sense of "flow" in performing their task. For delays of more than 1 second, indicate to the user that the computer is working on the problem, for example by changing the shape of the cursor.

Reply:
OK. I would still give it a name ;)

Reply:
Done! I've called it `MIN_LOADING_TIME`